package.json

```diff
@@ -18,33 +19,33 @@
     "compression": "^1.7.4",
     "express": "^4.16.4",
     "faye-websocket": "^0.11.3",
-    "graceful-fs": "^4.1.15",
-    "handlebars": "^4.5.3",
-    "http-proxy": "^1.17.0",
-    "ip-address": "^5.9.0",
-    "log4js": "^4.1.1",
-    "moment": "^2.24.0",
-    "morgan": "^1.9.1",
-    "nan": "^2.14.0",
-    "optimist": "0.6.1",
+    "graceful-fs": "^4.2.4",
+    "handlebars": "^4.7.6",
+    "http-proxy": "^1.18.1",
+    "ip-address": "^6.3.0",
+    "log4js": "^6.2.1",
+    "moment": "^2.26.0",
+    "morgan": "^1.10.0",
+    "nan": "^2.14.1",
+    "optimist": "github:rstudio/node-optimist#dbbadda31e53b63225a57e172a528b1ddc52be52",
     "pause": "0.1.0",
     "q": "^1.5.1",
-    "qs": "^6.7.0",
+    "qs": "^6.9.4",
     "send": "^0.17.0",
-    "shiny-server-client": "https://github.com/rstudio/shiny-server-client/archive/fb1aef1.tar.gz",
-    "sockjs": "^0.3.19",
+    "shiny-server-client": "github:rstudio/shiny-server-client#d63e91ef06ce3de04761912590decd63c719cdb5",
```

Comment:
Using this shorthand syntax was introduced in `npm@2.8.0`.

Should this be added to the engine definition?

Reply:
Location: https://github.com/npm/cli/blame/latest/changelogs/CHANGELOG-2.md#L3521

`package.json` npm engine definition: https://github.com/npm/npm/issues/449#issuecomment-720233